static/app/views/performance/traceDetails/content.tsx

```diff
@@ -94,14 +99,65 @@ class TraceDetailsContent extends React.Component<Props, State> {
   }
 
   renderTraceNotFound() {
-    const {meta} = this.props;
+    const {meta, traceEventView, traceSlug, organization, location} = this.props;
 
     const transactions = meta?.transactions ?? 0;
     const errors = meta?.errors ?? 0;
 
     if (transactions === 0 && errors > 0) {
+      const eventView = {...traceEventView};
+      eventView.query = `trace:${traceSlug} !event.type:transaction `;
+
       return (
-        <LoadingError message={t('The trace you are looking contains only errors.')} />
+        <React.Fragment>
+          <Alert type="error" icon={<IconInfo size="md" />}>
+            <ErrorLabel>
+              {t('The trace cannot be shown when all events are errors.')}
+            </ErrorLabel>
+            <DiscoverQuery
+              eventView={traceEventView}
```

Comment:
Should this be

```suggestion
              eventView={eventView}
```

to use the `eventView` you just created?

Additionally, consider giving it a more descriptive name like `errorsEventView` to clarify this `EventView`'s purpose.